### PR TITLE
Stellar: Show banners on send/confirm screens, show errors on sending

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers

### CODE REVIEW

File: shared/wallets/confirm-form/container.js [hunk 1]
```diff
@@ -7,21 +7,42 @@ import {connect, type TypedState} from '../../util/container'
...
     waitingKey: Constants.sendPaymentWaitingKey,
     yourUsername: state.config.username,
   }
 }
 
 const mapDispatchToProps = (dispatch, {navigateUp}) => ({
-  onBack: () => dispatch(navigateUp()),
-  onClose: () => dispatch(navigateUp()),
+  onBack: () => {
+    dispatch(WalletsGen.createClearErrors())
```
Comment (chrisnojima) [inline shared/wallets/confirm-form/container.js]: another pattern we have is doing this in the saga. you have a saga listener for navigate up and if its a stellar path you'd clear the error. https://github.com/keybase/client/blob/master/shared/actions/provision.js#L517

File: shared/wallets/send-form/body/container.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
 // @flow
 import Body from '.'
 import {compose, connect, setDisplayName, type TypedState} from '../../../util/container'
+import {bannerLevelToBackground} from '../../../constants/wallets'
 
-const mapStateToProps = (state: TypedState) => ({})
+const mapStateToProps = (state: TypedState) => ({
+  banners: (state.wallets.builtPayment.banners || []).map(banner => ({
```
Comment (chrisnojima) [inline shared/wallets/send-form/body/container.js]: likely better to do this in mergeProps so you don't thrash (even with the new empty array each time)

File: shared/wallets/confirm-form/container.js (hunk 1 above)
Comment (cjb) [reply shared/wallets/confirm-form/container.js]: Fixed via 03ef461

File: shared/wallets/send-form/body/container.js (hunk 2 above)
Comment (cjb) [reply shared/wallets/send-form/body/container.js]: Fixed via 4ba6b67

**Comment by cjb** [comment]
@chrisnojima Thanks! Ready for another look.

**Pull request merged**